src/framework/audio/internal/audiobuffer.cpp

```diff
@@ -120,9 +115,8 @@ void AudioBuffer::fillup()
     }
 
     while (sampleLag() < m_minSampleLag + FILL_OVER) {
-        m_source->setBufferSize(FILL_SAMPLES);
-        m_source->forward(FILL_SAMPLES);
-        push(m_source->data(), FILL_SAMPLES);
+        m_source->process(m_writeCache.data(), FILL_SAMPLES);
+        push(m_writeCache.data(), FILL_SAMPLES);
```

Comment:
Now, as we decided, data size multiply by FILL_SAMPLES, not needs m_writeCache

Reply:
yep, enough to pass a field of memory in m_data